test/test_torch.py

```diff
@@ -1815,6 +1826,11 @@ def test_mul(self):
             a2 = torch.tensor([True, False, True, False], dtype=torch.bool, device=device)
             self.assertEqual(a1 * a2, torch.tensor([True, False, False, False], dtype=torch.bool, device=device))
 
+            if device == 'cpu':
```

Comment:
super nit: As separate test or out of the loop scope

Reply:
I would prefer keeping it all in one test even with 'hacks' like this - its much easier to support these tests when they are in the same place. We already have test_cuda which partially tests the same thing but on CUDA. Ideally, once someone changes particular functionality he should update 1 test only.

Reply:
Also, this is a convenient way of seeing what is supported and what is not.